### DISCUSSION TITLE
How do i set size and location with queryparameters?

### DISCUSSION THREAD

**Discussion by nikkologen** [SixLabors/ImageSharp.Web#302] [Q&A]
https://localhost:44315/image.jpg?width=200&height=100&quality=80&rxy=0.1,0.5&rmode=manual
When trying to crop using the manual mode i get an: InvalidOperationException: Manual resizing requires a target location and size.

but i cant figure out how to set target location and size.

Any advice?

**Comment by stefannikolei** [comment]
It seems to me that you can not provide those parameters. I will check it and perhaps create an PR to add them.

**Comment by stefannikolei** [comment]
@nikkologen this will be a feature in the next version

**Reply by nikkologen** [reply]
Thank you. Much aprechiated.
